_episodes/02-loop.md

```diff
@@ -141,6 +141,14 @@ for variable in collection:
 ~~~
 {: .language-python}
 
+> ## What is the ":" for?
+>
+> One thing you may have noticed is that our `for` statement ends with a `:`. You actually will see the `:` regularly in Python, and
+> it actually can serve two functions. The first is indexing, which you became familiar with in the previous episode. The second, as is
```

Comment:
1. word "actually" appears twice in this sentence.
2. We should not draw student's attention from the current topic. Reminding is good but only once you finished explaining current topic. We should explain the purpose of colons in this particular case.